tests/cli/commands/test_task_command.py

```diff
@@ -62,6 +62,11 @@ class TestCliTasks(unittest.TestCase):
     def setUpClass(cls):
         cls.dagbag = DagBag(include_examples=True)
         cls.parser = cli_parser.get_parser()
+        clear_db_runs()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        clear_db_runs()
```

Comment:
I think this should be `tearDown` not `tearDownClass` because we don’t want the DAG runs to carry over to other tests in the same class.